pdns/README-dnsdist.md

```diff
@@ -1385,6 +1385,7 @@ instantiate a server with additional parameters
     * `setServerPolicyLua(name, function)`: set server selection policy to one named 'name' and provided by 'function'
     * `showServerPolicy()`: show name of currently operational server selection policy
     * `newServerPolicy(name, function)`: create a policy object from a Lua function
+    * `setServFailWhenNoServer(bool)`: if set, returns a ServFail when no server are available instead of dropping the query (the default)
```

Comment:
'if set, return a ServFail when no servers are available, instead of the default behaviour of dropping the query'